doc/appendix-de-kassensichv/data-structures/data-structures.md

```diff
@@ -23,55 +23,55 @@ If you need to provide customer data in your request, you can send it in via the
-| `RefClosingNr` | `integer` | If the reference you want to add is of type (RefType) "Transaktion", you must provide data for the field `RefClosingNr`. |  It should provide the the referenced cashpoint daily closing number of the referenced object. Please send it via `ftReceiptCaseData` in JSON format by adding the key value pair `RefClosingNr` e.g. `"ftReceiptCaseData":"{ ..., "RefClosingNr":1091029, ... }"`. Starting from version 1.3.6, the closing numbers to reference can be obtained from the `ftStateData` field of daily closing receipts' responses (via the `DailyClosingNumber` JSON element).  | 1.3 |
-| `RefReceiptId` | `string` | If the reference you want to add is of type (RefType) "Transaktion", you must provide data for the field `RefReceiptId`. |  It should contain the value of ´ftReceiptIdentification´ of the referenced object. Please send it via `ftReceiptCaseData` in JSON format by adding the key value pair `RefReceiptId` e.g. `"ftReceiptCaseData":"{ ..., "RefReceiptId":"UAUUA1112#20200211-112430", ... }"`. | 1.3 |
+| `RefClosingNr` | `integer` | If the reference you want to add is of type (RefType) "Transaktion", you must provide data for the field `RefClosingNr`. |  It should provide the referenced cashpoint daily closing number of the referenced object. Please send it via `ftReceiptCaseData` in JSON format by adding the key value pair `RefClosingNr` e.g. `"ftReceiptCaseData":"{ ..., "RefClosingNr":1091029, ... }"`. Starting from version 1.3.6, the closing numbers to reference can be obtained from the `ftStateData` field of daily closing receipts' responses (via the `DailyClosingNumber` JSON element).  | 1.3 |
+| `RefReceiptId` | `string` | If the reference you want to add is of type (RefType) "Transaktion", you must provide data for the field `RefReceiptId`. | It should contain the value of ´ftReceiptIdentification´ of the referenced object. Please send it via `ftReceiptCaseData` in JSON format by adding the key value pair `RefReceiptId` e.g. `"ftReceiptCaseData":"{ ..., "RefReceiptId":"UAUUA1112#20200211-112430", ... }"`. | 1.3 |
 
 ### Charge Items Entry
 
-In the general description the field `ftChargeItemCaseData` is described as optional. However for the German market the content of this field is not always optional.
+In the general description, the field `ftChargeItemCaseData` is described as optional. However, for the German market, the content of this field is not always optional.
 
-For some special cases we need you to transmit data within the field `ftChargeItemCaseData` that is later needed for the DSFinV-K export. The following table describes when and how you have to fill them.
+For some cases, we need you to transmit data within the field `ftChargeItemCaseData` that is required later for the DSFinV-K export. The following table describes when and how you have to fill them.
```

Comment:
For some cases, it is needed to transmit data within the field --------, that is required later for the DSFinV-K export. The following table describes how and when these need to be filled.